src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
+      <div class="star"></div>
```

Comment:
You should add some 'air' between these sibling blocks of HTML to simplify reading. But don't add them between parent and child elements. Also, it's worth considering adding some alternative content for these divs, as they currently don't provide any information about their content.